Formula/trailscraper.rb

```diff
@@ -3,9 +3,8 @@ class Trailscraper < Formula
 
   desc "Tool to get valuable information out of AWS CloudTrail"
   homepage "https://github.com/flosell/trailscraper"
-  url "https://github.com/flosell/trailscraper/archive/0.6.1.tar.gz"
-  sha256 "9ca2535d397282ffc09e5ca05af19bc36dd3da02235df963c3016b9f2dc4d151"
-  license "Apache-2.0"
+  url "https://github.com/flosell/trailscraper/archive/0.6.4.tar.gz"
+  sha256 "4cc20f257c8167ca0f48aab3f0f2e29e60bd8ab5316d3f16e88a35e5eab039cc"
```

Comment:
```suggestion
  sha256 "4cc20f257c8167ca0f48aab3f0f2e29e60bd8ab5316d3f16e88a35e5eab039cc"
  license "Apache-2.0"
```

Reply:
I assume the `license` tag was removed in error?

The tag was only recently rolled out broadly so maybe one of the scripts you used doesn’t know this is a thing 🤔

Reply:
Whoops, sorry about that! Thanks for catching it! And you are right, my scripting didn't know this was a thing :) I'll fix that!